src/ts/util.ts

```diff
@@ -2,10 +2,9 @@
 
 import {URL} from 'url';
 
-// tslint:disable-next-line no-var-requires
 const crypto = require('crypto');
 
-export function hash(...urls: Array<URL | undefined>): string {
+export function hash(...urls: (URL | undefined)[]): string {
```

Comment:
If array is different type of element its preferable to use Array<>. https://typescript-eslint.io/rules/array-type/#options. Array rule needs to be revisited.

Reply:
I reverted this change and disabled the array-type ESLint rule for now.